honeybee_grasshopper_energy/src/HB Blend ProgramTypes.py

```diff
@@ -0,0 +1,54 @@
+# Honeybee: A Plugin for Environmental Analysis (GPL)
+# This file is part of Honeybee.
+#
+# Copyright (c) 2019, Ladybug Tools.
+# You should have received a copy of the GNU General Public License
+# along with Honeybee; If not, see <http://www.gnu.org/licenses/>.
+# 
+# @license GPL-3.0+ <http://spdx.org/licenses/GPL-3.0+>
+
+"""
+Create a ProgramType object by blending other ProgramTypes together using a weighted
+average based on program ratios.
+-
+
+    Args:
+        _name: Text string for the new blended program type name.
+        _programs: A list of ProgramType objects that will be averaged
+            together to make a new ProgramType.
+        _ratios_: A list of fractional numbers with the same length as the input
+            programs that sum to 1. These will be used to weight each of the
+            ProgramType objects in the resulting average. If None, the input
+            program objects will be weighted equally. Default: None.
+    
+    Returns:
+        program: A ProgramType object that's a weighted average between the
+            input _programs.
+"""
+
+ghenv.Component.Name = "HB Blend ProgramTypes"
+ghenv.Component.NickName = 'BlendPrograms'
+ghenv.Component.Message = '0.1.0'
+ghenv.Component.Category = "Energy"
+ghenv.Component.SubCategory = '0 :: Basic Properties'
```

Comment:
Because that is too abstract and implies that all of the other tabs (constructions, schedules, loads) are not properties.  For this reason would sooner call the tab '0 :: Basic' or '0 :: Essential' or '0 :: Main' rather than '0 :: Properties'.